enterprise/config/buildbuddy.local.yaml

```diff
@@ -13,8 +13,58 @@ storage:
     root_directory: /tmp/buildbuddy_enterprise
   enable_chunked_event_logs: true
 cache:
+  # Sample flow if migration disk -> pebble
+
+  # Begin double writing, and still return data from disk
   disk:
     root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_src: true
```

Comment:
Overall, I like this proposal. Siggi's comment about same-cache-type migration needs consideration.
We have some same-cache-type migrations in mind that we want to do (e.g. moving existing pebble cache to smaller disks) and we might have more in the future.